src/test/testSetup/controller.testSetup.ts

```diff
@@ -486,6 +490,62 @@ export const getTestSetups = () => {
         shouldLoadDataFromCacheOnQuickPickOpen: false,
       });
     },
+    search10: () => {
+      stubComponents(sandbox);
+      stubComponentsForSearch(sandbox, {
+        isInitialized: true,
+      });
+      return stubMultiple(
+        [
+          {
+            object: quickPick,
+            method: "loadItems",
+          },
+          {
+            object: quickPick,
+            method: "show",
+          },
+          {
+            object: quickPick,
+            method: "setText",
+          },
+          {
+            object: config,
+            method: "fetchShouldQuickPickOpenFromSelection",
+            returns: true,
+          },
```

Comment:
Could not stub `vscode.window.activeTextEditor`.
So I could not write a test when `vscode.window.activeTextEditor` is undefined 😭 

At that time I had to write code like this.
```js
{
  object: vscode,
  method: "window",
  returns: ({ activeTextEditor: undefined })
},
```